### PR TITLE
[CFM-874] notify user that was demoted from GA to GM - template update required

### PR DESCRIPTION
**Pull request by anvmn**
## This PR is dependent on #1059 ,#1061

https://issuetracker.amplexor.com/jira/browse/CFM-874

### CODE REVIEW

**Comment by anvmn** [comment]
selection_003

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc [hunk 1]
```diff
@@ -149,6 +149,27 @@ function c4m_user_notifications_notify_admins_admin_granted($gid, $uid) {
...
+ * Helper function.
+ *
+ * Sends mail notification, when group admin is demoted to regular member.
+ *
+ * @param int $gid
+ *   Group id.
+ * @param int $uid
+ *   User id.
+ */
+function _c4m_user_notify_user_admin_revoked($gid, $uid) {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc]: @anvmn 
replace "revoke" with "demoted"

**Comment by anvmn** [comment]
This PR is dependent on https://github.com/capacity4dev/capacity4more/pull/1061, where message field_related_user is introduced.

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc (hunk 1 above)
Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc]: @ordavidil 
Done

File: project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module [hunk 2]
```diff
@@ -76,6 +77,18 @@ function _c4m_message_format_welcome_to_group_from_field(&$message, $message_ent
 }
 
 /**
+ * Formats 'c4m_group_admin_demoted' message From field, as defined at template.
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module]: @anvmn 
Can you please write the ID of the template ?
thanks

Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module]: @ordavidil 
Do you want this at code, as part of documentation?

**Comment by anvmn** [comment]
out

**Comment by anvmn** [comment]
CFM 873:
selection_059

**Comment by anvmn** [comment]
## Replaced by #1189

**Pull request closed without merging**